### PR TITLE
FD-24: Establish foundational Flask app code.

### PR DESCRIPTION
**Pull request by rwilkinson3**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+flask==2.2.3
+flask-cors==3.0.10
+mysql-connector-python==8.0.32
```
Comment (msherry2) [inline requirements.txt:3]: I needed to add `sshtunnel==0.4.0`

**Review by msherry2 — CHANGES_REQUESTED** [review]
`sshtunnel` needs to be added as a requirement.

Other note: front end will need the base urls from the CONFIG section of `file_list.json`. I think it makes sense to send it along with the response, but that might fall under the `# TODO Determine the response structure we need for the frontend`. So up to you if you want to include it now or not. 
It would also be good to add some of the README info to Confluence, but that can happen at any time.

File: requirements.txt (hunk 1 above)
Comment (rwilkinson3) [reply requirements.txt:3]: It was determined in FD's 2/27 meeting that BigQuery will be the only source of the monthly game usage stats, so I've removed the MySQL and sshtunnel dependencies. BigQuery dependencies will need to be added in FD-25.

Yes, I expect the API will automatically combine the URL bases with the file paths in the response.

**Review by msherry2 — APPROVED** [review]
works great, thanks!

**Pull request merged**